.prettierrc

```diff
@@ -4,5 +4,5 @@
   "tabWidth": 2,
   "semi": true,
   "singleQuote": true,
-  "endOfLine": "lf"
+  "endOfLine": "crlf"
```

Comment:
let us not change this configuration

Reply:
Actually Mahmoud suggested this change to solve prettier & eslint problems in my settings & it worked.. but I'll try to get back as you suggested

Reply:
I got it back to 'lf' [&](url) errors have came back too... 
![image](https://user-images.githubusercontent.com/63739334/89438219-39302200-d751-11ea-864b-5391884b2fb4.png)